src/lib/steps/rollup.ts

```diff
@@ -23,37 +23,31 @@ export interface RollupOptions {
  */
 export async function rollup(opts: RollupOptions): Promise<void> {
   log.debug(`rollup (v${ROLLUP_VERSION}) ${opts.entry} to ${opts.dest} (${opts.format})`);
-
-  const globals = {
-    // default externals for '@angular/*' and 'rxjs'
-    ...ROLLUP_GLOBALS,
-    // external symbols passed from the user's ng-package.json
-    ...opts.externals,
+  const embedded: string[] = opts.embedded || [];
+  const namedExternals = {
+    'tslib': 'tslib',
```

Comment:
might also make sense to add some other defaults that are widely used example, `ionic-angular`, `lodash`, `moment`, `date-fns`, `react` etc..